mongo/integration/mtest/opmsg_deployment.go

```diff
@@ -21,23 +21,31 @@ import (
 )
 
 const (
-	serverAddress                = address.Address("localhost:27017")
-	maxDocumentSize       uint32 = 16777216
-	maxMessageSize        uint32 = 48000000
-	maxBatchCount         uint32 = 100000
-	sessionTimeoutMinutes uint32 = 30
+	serverAddress          = address.Address("localhost:27017")
+	maxDocumentSize uint32 = 16777216
+	maxMessageSize  uint32 = 48000000
+	maxBatchCount   uint32 = 100000
 )
 
 var (
+	sessionTimeoutMinutes = func() *uint32 {
```

Comment:
Optional: Consider making `sessionTimeoutMinutes` a `uint32` variable and then using address-of when you need a pointer.